test/postControllerTest.js

```diff
@@ -0,0 +1,187 @@
+/* global it, describe */
+const express = require('express');
+const request = require('supertest');
+const postController = require('../controllers/postController');
+const sinon = require('sinon');
+const postService = require('../services/postService');
+const assert = require('assert');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(express.static('../public'));
+
+app.use('', postController);
+
+describe('postController', () => {
+  let stubGetAllPosts = sinon.stub();
+  let stubGetOnePost = sinon.stub();
+  let stubCreateNewPost = sinon.stub();
```

Comment:
What's the point of assigning a value if it's going to be overwritten in `beforeEach`?

Reply:
Wouldn't the stubs not be accessible because both let and const are block scoped? I don't know how else to create them without making the stubs global.

Reply:
you can change `let stubCreateNewPost = sinon.stub();` to `let stubCreateNewPost;`

Alternatively, you can remove these variable declarations, and in `beforeEach`, change `stubGetOnePost = sinon.stub(postService, 'getOnePost').resolves('expected');` to `sinon.stub(postService, 'getOnePost').resolves('expected');` and in `afterEach`, change `stubGetOnePost.restore();` to `postService.getOnePost.restore()`